llvm/test/Verifier/branch-weight.ll

```diff
@@ -0,0 +1,39 @@
+; Test MD_prof validation
+
+; RUN: split-file %s %t
+; RUN: opt -passes=verify %t/valid.ll --disable-output
+; RUN: not opt -passes=verify %t/invalid1.ll --disable-output 2>&1 | FileCheck %s
+; RUN: not opt -passes=verify %t/invalid2.ll --disable-output 2>&1 | FileCheck %s
```

Comment:
Seems a bit ad-hoc to share the rely on the check annotation of the source file. I'm unsure how this is done normally, so maybe this is fine(?).
I would probably pass `%t/invalid1.ll` and `%t/invalid2.ll` to `FileCheck` instead of `%s` in both cases.

Reply:
The usual way to do this is passing `%s` to FileCheck but using a different `--check-prefix` for each file. But given that the error message is the same here, this should be fine as-is...